openedx/core/djangoapps/catalog/utils.py

```diff
@@ -31,6 +32,9 @@
 from openedx.core.djangoapps.oauth_dispatch.jwt import create_jwt_for_user
 from openedx.core.lib.edx_api_utils import get_api_data
 
+if TYPE_CHECKING:
```

Comment:
anything in a `if TYPE_CHECKING` block isn't actually imported, it's just imported during linter/mypy type checks.